src/Group.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Owncloud\OcisPhpSdk;
+
+use OpenAPI\Client\Model\User;
+
+class Group
+{
+    /**
+     * @param string $id
+     * @param string $description
+     * @param string $displayName
+     * @param array<int,string> $groupTypes
+     * @param array<int,User> $members
+     * @param array<int,string> $membersodatabind
+     */
+    public function __construct(
+        private string $id,
+        private string $description,
+        private string $displayName,
+        private array $groupTypes,
+        private array $members,
+        private array $membersodatabind,
+    ) {
+    }
+
+    /**
+     * @return string
+     */
+    public function getId(): string
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return $this->description;
+    }
+
+    /**
+     * @return string
+     */
+    public function getDisplayName(): string
+    {
+        return $this->displayName;
+    }
+
+    /**
+     * @return array<int,string>
+     */
+    public function getGroupTypes(): array
+    {
+        return $this->groupTypes;
+    }
+
+    /**
+     * @return array<int,User>
```

Comment:
let's have an own Users class here that can return
- id
- display name
- mail
- on_premises_sam_account_name